src/components/color-picker/color-picker.component.ts

```diff
@@ -17,7 +17,8 @@ let uniqueId = 0;
 @Component({
     selector: 'ux-color-picker',
     exportAs: 'ux-color-picker',
-    templateUrl: 'color-picker.component.html'
+    templateUrl: 'color-picker.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
Looks like I forgot to click the button - this one.

Reply:
removed and seems to work the same